src/main/java/com/coreos/jetcd/EtcdClientBuilder.java

```diff
@@ -100,7 +104,7 @@ public EtcdClientBuilder setPassword(ByteString password) {
      * @return this builder
      * @throws NullPointerException if password is null
      */
-    public EtcdClientBuilder setNameResolverFactory(AbstractEtcdNameResolverFactory nameResolverFactory){
+    public EtcdClientBuilder setNameResolverFactory(AbstractEtcdNameResolverFactory nameResolverFactory) {
```

Comment:
Tks a lot for your reviews, and I will fix the codes.  I think we should use `public` here, cause there are several classes which extend `AbstractEtcdNameResolverFactory`, and sometimes  users may want to use their own implementations, it would be better if we expose this out for them.